jqm-all/jqm-engine/src/main/java/com/enioka/jqm/tools/QueuePoller.java

```diff
@@ -318,15 +318,15 @@ public synchronized void run() // sync: avoid race condition on run when restart
                 }
                 else
                 {
-                    jqmlogger.error("Queue poller has failed! It will stop.", e);
+                    jqmlogger.error("Queue poller has failed! It will stop (RuntimeException).", e);
```

Comment:
Why?

Reply:
I wanted to know which kind of exception was caught as there is two catch blocks with same log message. Maybe not useful, I can revert this change.